github-actions/slash-commands/lib/commands/rerun-circleci.ts

```diff
@@ -108,8 +108,8 @@ class CircleCiClient {
       requestInit['body'] = JSON.stringify(body);
     }
 
-    const response = await fetch.default(url, requestInit);
-    const responseJson = await response.json();
+    const response = await fetch(url, requestInit);
+    const responseJson = (await response.json()) as any;
```

Comment:
Can we add a TODO here to define the types we expect for this json?

Reply:
Done, also added a type.